.eslintrc.json

```
@@ -0,0 +1,27 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides": [
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "ignorePatterns": ["dist/*"],
+    "rules": {
+      "@typescript-eslint/consistent-type-imports": ["error"],
+      "no-restricted-imports": ["error", {
+        "patterns": ["*.ts"]
+      }]
```

```suggestion
        "@typescript-eslint/consistent-type-imports": ["error"],
        "no-restricted-imports": ["error", {
            "patterns": ["*.ts"]
        }]
```
Consistent indentation :)